metadata-ingestion/src/datahub/ingestion/source/superset.py

```diff
@@ -123,6 +148,9 @@ class SupersetConfig(
         description="Can be used to change mapping for database names in superset to what you have in datahub",
     )
 
+    class Config:
+        extra = "allow"
```

Comment:
let's add a comment here clarifying that this is required to allow preset configs to get parsed